legal.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: page
+title: Privacy Policy
+background: grey
+---
+
+<div class="col-lg-12 text-center">
+	<h2 class="section-heading text-uppercase">Privacy Policy</h2>
+</div>
+
+This Privacy Policy describes how your personal information is collected, used, and shared when you visit {{ site.title }} (the “Site”).
+
+**PERSONAL INFORMATION WE COLLECT**
+
+{% if site.analytics.google %}
+
+Automatically Collected (Google Analytics):
+
+When you visit the Site, we automatically receive information about your device from your browser, such as your IP address. As you browse the Site, we also collect information about how you interact with the Site. We refer to this automatically-collected information as “Device Information”.
+
+We collect Device Information using cookies. “Cookies” are data files that are placed on your device. For more information about cookies and how to disable them, visit http://www.allaboutcookies.org.
+
+We do this using Google Analytics: <https://www.google.com/intl/en/policies/privacy/>.
```

Comment:
Does the site have google analytics set up already, or is this something I could choose to add in the future?

Reply:
> Does the site have google analytics set up already, or is this something I could choose to add in the future?

No. it's the content on the agency theme, you can ignore it. and delete it

Reply:
Great, will do